--- a/CMakeLists.txt	2026-09-02 19:40:43
+++ b/CMakeLists.txt	2026-09-02 23:25:00
@@ -2,7 +2,7 @@
   project(llvm-cbe)
   set (USE_SYSTEM_LLVM 1)
   cmake_minimum_required(VERSION 3.20.0)
-  find_package(LLVM 22.1 REQUIRED CONFIG)
+  find_package(LLVM REQUIRED CONFIG)
   message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
   message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
   message(STATUS "LLVM_INCLUDE_DIRS: ${LLVM_INCLUDE_DIRS}")
